languagetool-language-modules/zh/src/main/java/org/languagetool/tokenizers/zh/ChineseWordTokenizer.java

```diff
@@ -19,42 +19,25 @@
 package org.languagetool.tokenizers.zh;
 
 import java.io.IOException;
-import java.io.InputStream;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.List;
 
-import org.ictclas4j.segment.SegTag;
-import org.languagetool.JLanguageTool;
-import org.languagetool.databroker.ResourceDataBroker;
+import com.hankcs.hanlp.model.perceptron.PerceptronLexicalAnalyzer;
+import com.hankcs.hanlp.seg.common.Term;
+
 import org.languagetool.tokenizers.Tokenizer;
 
-import cn.com.cjf.CJFBeanFactory;
-import cn.com.cjf.ChineseJF;
 
 public class ChineseWordTokenizer implements Tokenizer {
 
-  private SegTag seg;
-  private ChineseJF chinesdJF;
+  private PerceptronLexicalAnalyzer seg;
 
   private void init() {
-    if (chinesdJF == null) {
-      chinesdJF = CJFBeanFactory.getChineseJF();
-    }
     if (seg == null) {
-      ResourceDataBroker dataBroker = JLanguageTool.getDataBroker();
-      try (InputStream coreDictIn = dataBroker.getFromResourceDirAsStream("/zh/coreDict.dct");

```

Comment:
So these files are not needed anymore, is that correct? If so, please remove them from git (or did you? I didn't see their removal being part of the PR).

Reply:
Mhh, maybe I just commented on an old PR? Not sure, if so, just ignore this...